test/utils/miscellaneous.spec.ts

```diff
@@ -89,3 +90,59 @@ describe('getToken', () => {
     })
   })
 })
+
+describe('isAnOrderId', () => {
+  it('should be an orderId', () => {
+    const text =
+      '0x405bd0278c11399f84f10e19fb9b45123996e7d0a68a60ddebc3b9581576b484ff714b8b0e2700303ec912bd40496c3997ceea2b614b17d9'
+
+    const result = isAnOrderId(text)
+
+    expect(result).toBe(true)
+  })
+
+  it('not be an orderId', () => {
```

Comment:
uber-nitpicking: The name of the `it` is not completly coherent with the other one. 

This one should be according to the other `shouldn't be an order id`, however, I would simplify all of them to
`Is orderId`, `Is not orderId`